utils/validator_info.go

```diff
@@ -22,179 +24,184 @@ func PeerIndex(peers *models.PeersResponse, nodeID string) int {
 	}
 	return -1
 }
-func GetDate(unixTime string) string {
+func GetDate(unixTime string) (string, error) {
 	// Date in Unix Format
-	unixDateInt, _ := strconv.ParseInt(unixTime, 10, 64)
+	unixDateInt, err := strconv.ParseInt(unixTime, 10, 64)
+	if err != nil {
+		return "", err
+	}
 	// UnixDate in Time format struct
 	dateFTime := time.Unix(unixDateInt, 0)
-	return strings.Split(dateFTime.String(), " -")[0]
+	return strings.Split(dateFTime.String(), " -")[0], nil
 }
-func getDuration(startTime string, endTime string) string {
-	start, _ := time.Parse("2006-01-02 15:04:05", startTime)
-	end, _ := time.Parse("2006-01-02 15:04:05", endTime)
+
+func getDuration(startTime string, endTime string) (string, error) {
+	start, err := time.Parse(timeLayout, startTime)
```

Comment:
Maybe I didn't describe well my thought. Instead of defining `timeLayout` why don't we use in this line the `time.UnixDate`? This date is already defined in the `time` package in the `UnixDate` varibale and we can directly use it instead of defining it again.